src/condor_io/condor_auth_ssl.cpp

```diff
@@ -1922,25 +1925,67 @@ SSL_CTX *Condor_Auth_SSL :: setup_ssl_ctx( bool is_server )
 
 	// Only load the verify locations if they are explicitly specified;
 	// otherwise, we will use the system default.
-    if( (cafile || cadir) && SSL_CTX_load_verify_locations_ptr( ctx, cafile, cadir ) != 1 ) {
+
+    if (cafile) {
+        StringList cafile_list(cafile, ",");
+        cafile_list.rewind();
+        char *ca;
+        while ((ca = cafile_list.next())) {
+            std::string ca_str(ca);
+            trim(ca_str);
```

Comment:
trim unnecessary?  pretty sure that stringlist will always trim.

Reply:
Good point.

Reply:
Harmless but unnecessary.